weave-js/src/components/RadixTooltip/Tooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+
+import * as RadixTooltip from './index';
+
+export type TooltipProps = {
+  trigger: React.ReactNode;
+  content: React.ReactNode | string;
+  /**
+   * This determines if the trigger and content components should be wrapped in
+   * RadixTooltip.Trigger and RadixTooltip.Content, respectively. Otherwise, it
+   * will just assume props passed in are properly wrapped in RadixTooltip anatomy.
+   * Having this option allows a bit more flexibilty and less prop passing.
+   *
+   * Setting it to false will tell the component to not apply the RadixTooltip.Trigger
+   * and RadixTooltip.Content around the props.
+   */
+  nowrap?: boolean;
```

Comment:
I notice that the simple cases are passing an icon as trigger and text as content; however, I'm also seeing many complicated content components [like this one](https://github.com/wandb/core/blob/64fba8aed935ef31aa9714ebfc6810c4b59548f6/frontends/app/src/components/TeamSettings/TeamSettings.tsx#L892-L929).

What are people's thoughts on this?

example PR of simple usage: https://github.com/wandb/core/pull/20851

Reply:
tooltips are generally meant to be simple text bubbles that provide context/description on hover. for richer content, it's more appropriate to use radix's Popover.

Radix docs for the two components

[Tooltip](https://www.radix-ui.com/primitives/docs/components/tooltip): "A popup that displays information related to an element when the element receives keyboard focus or the mouse hovers over it."

[Popover](https://www.radix-ui.com/primitives/docs/components/popover): "Displays rich content in a portal, triggered by a button."

The aria roles radix uses for tooltip vs popover are [tooltip](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) and [dialog](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role) respectively. Those docs should also help provide some insight on the accessibility implications of using one vs the other.

That all being said, I don't know if we'll have a hard time making sure everyone knows to use Popover because I get a feeling most folks will reach for tooltip by default

Reply:
Based on conversations I've had with folks (and the numerous different tooltips), it's going to be pretty hard to enforce especially since design uses the "tooltip" language. General theme is that people just want to have a common component to use because we have too many. i am hoping to keep scope just for that - get a simple tooltip going with an escape hatch lol